media/js/privacy/privacy-protocol.js

```diff
@@ -19,3 +19,16 @@
         buttons[i].setAttribute('data-close', closeText);
     }
 })();
+
+if (window.location.href.includes('cookies')) {
+    window.onload = function() {
+        'use strict';
+        
+        var cookieButton = document.querySelector('#user-choices button');
+        var cookieText = document.getElementById('expand-formatparagraphssummary-2');
+        
+        cookieButton.setAttribute('aria-expanded', 'true');
```

Comment:
These lines will throw errors should the selectors above not be found, and since the HTMl on this page is generated via markdown it could be a little brittle.

To avoid throwing errors should something go wrong, I'd suggest adding a check that both selectors have found a match:

```javascript
if (cookieButton && cookieText) {
    cookieButton.setAttribute('aria-expanded', 'true');
    cookieText.classList.remove('is-closed');
    cookieText.setAttribute('aria-hidden', 'false');
}
```